cluster/src/main/java/com/hartwig/pipeline/transfer/google/RetryHttpInitializerWrapper.java

```diff
@@ -0,0 +1,71 @@
+package com.hartwig.pipeline.transfer.google;
+
+import java.util.logging.Logger;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.http.HttpBackOffIOExceptionHandler;
+import com.google.api.client.http.HttpBackOffUnsuccessfulResponseHandler;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpUnsuccessfulResponseHandler;
+import com.google.api.client.util.ExponentialBackOff;
+import com.google.api.client.util.Sleeper;
+import com.google.common.base.Preconditions;
+
+/**
```

Comment:
was this class copied from somewhere (javadoc?)?

Reply:
Yeah. Grabbed straight from the documentation. I'll tidy.